data/creaturescripts/scripts/login.lua

```diff
@@ -1,4 +1,8 @@
 function onLogin(player)
+	if configManager.getBoolean(configKeys.MAINTENANCE_MODE) and player:getGroup():getId() < ACCOUNT_TYPE_TUTOR then
```

Comment:
Comparing group ID with account type won't work

Reply:
Doing this in an onLogin script will also trigger other onLogin scripts, and you won't know which one ran first (and potentially already executed code by the time it hits this one)